src/Clustering/GMUClusterManager.h

```diff
@@ -45,12 +50,19 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface GMUClusterManager : NSObject<GMSMapViewDelegate>
 
-/** Initializer with the |mapView|, |algorithm| and |renderer|. */
-- (instancetype)initWithMap:(GMSMapView *)mapView
-                  algorithm:(id<GMUClusterAlgorithm>)algorithm
-                   renderer:(id<GMUClusterRenderer>)renderer;
+/**
+ * The default initializer is not available. Use initWithMap:algorithm:renderer instead.
+ */
+- (instancetype)init NS_UNAVAILABLE;
+
+/**
+ * Returns a new instance of the GMUClusterManager class defined by it's |algorithm| and |renderer|.
+ */
+- (instancetype)initWithMap:(GMSMapView *)mapView algorithm:(id<GMUClusterAlgorithm>)algorithm renderer:(id<GMUClusterRenderer>)renderer NS_DESIGNATED_INITIALIZER;
```

Comment:
We are trying to keep the coding style to the same as the GoogleMaps headers. In general we want lines to be within 100 character limit for readability purposes (e.g code fits within 1 screen width).

Until we all have agreed to allow inline header declaration, we'd like to keep it this way. Thanks for the understanding.